src/dialog/overflow-menu/overflow-menu-option.component.ts

```diff
@@ -88,6 +106,8 @@ export class OverflowMenuOption implements AfterViewInit {
 	// change after checked errors
 	public title = null;
 
+	protected _target?: Target;
+
```

Comment:
I don't think we need this?

```suggestion
```

Reply:
True. But went back and got to use it again. What you do think of it now?